scipy/signal/signaltools.py

```diff
@@ -892,41 +892,90 @@ def _prod(iterable):
     return product
 
 
-def _fftconv_faster(x, h, mode):
+def _conv_ops(x_shape, h_shape, mode):
     """
-    See if using `fftconvolve` or `_correlateND` is faster. The boolean value
-    returned depends on the sizes and shapes of the input values.
+    Find the number of operations required for direct/fft methods of
+    convolution
+
+    The direct operations were recorded by making a dummy class to record the
+    number of multplications. The number of additions is always less than or
+    equal to the number of multiplications. More detail visible at [1].
```

Comment:
Where is the reference for this citation?

Reply:
I've stripped it out. I planned to link to @drorspei's [`conv-complexity.ipynb` notebook][1], but it doesn't add much given the implementation and a description of what that notebook does.

[1]:https://github.com/drorspei/misc/blob/master/notebooks/conv-complexity.ipynb